### PR TITLE
De-XHRify CORS guide

### PR DESCRIPTION
**Pull request by wbamberg**
Part of https://github.com/mdn/content/issues/31010. I've deliberately kept this very focused on XHR -> fetch, although I do think this guide could use a refresh.

### CODE REVIEW

File: files/en-us/web/http/cors/index.md [hunk 1]
```diff
@@ -269,25 +277,24 @@ However, if the request is one that triggers a preflight due to the presence of
 
 > **Note:** When making credentialed requests to a different domain, third-party cookie policies will still apply. The policy is always enforced regardless of any setup on the server and the client as described in this chapter.
 
-The most interesting capability exposed by both {{domxref("XMLHttpRequest")}} or [Fetch](/en-US/docs/Web/API/Fetch_API) and CORS is the ability to make "credentialed" requests that are aware of [HTTP cookies](/en-US/docs/Web/HTTP/Cookies) and HTTP Authentication information. By default, in cross-origin `XMLHttpRequest` or [Fetch](/en-US/docs/Web/API/Fetch_API) invocations, browsers will **not** send credentials. A specific flag has to be set on the `XMLHttpRequest` object or the {{domxref("Request")}} constructor when it is invoked.
+The most interesting capability exposed by both {{domxref("fetch()")}} or {{domxref("XMLHttpRequest")}} and CORS is the ability to make "credentialed" requests that are aware of [HTTP cookies](/en-US/docs/Web/HTTP/Cookies) and HTTP Authentication information. By default, in cross-origin `fetch()` or `XMLHttpRequest` calls, browsers will **not** send credentials.
```
Comment (teoli2003) [inline files/en-us/web/http/cors/index.md:280]: NIT: (I don't like bold for emphasis, italics is better)
```suggestion
The most interesting capability exposed by both {{domxref("fetch()")}} or {{domxref("XMLHttpRequest")}} and CORS is the ability to make "credentialed" requests that are aware of [HTTP cookies](/en-US/docs/Web/HTTP/Cookies) and HTTP Authentication information. By default, in cross-origin `fetch()` or `XMLHttpRequest` calls, browsers will _not_ send credentials.
```

**Review by teoli2003 — APPROVED** [review]
One nit that I will commit, else it looks good. Thank you!

File: files/en-us/web/http/cors/index.md [hunk 2]
```diff
@@ -80,15 +80,16 @@ A _simple request_ is one that **meets all the following conditions**:
 >
 > No other browsers implement these extra restrictions because they're not part of the spec.
 
-For example, suppose web content at `https://foo.example` wishes to invoke content on domain `https://bar.other`. Code of this sort might be used in JavaScript deployed on `foo.example`:
+For example, suppose web content at `https://foo.example` wishes to fetch JSON content from domain `https://bar.other`. Code of this sort might be used in JavaScript deployed on `foo.example`:
 
 ```js
-const xhr = new XMLHttpRequest();
-const url = "https://bar.other/resources/public-data/";
+const fetchPromise = fetch("https://bar.other");
```
Comment (wbamberg) [inline files/en-us/web/http/cors/index.md:86]: hm, should this include `mode: "cors"`?

Comment (wbamberg) [reply files/en-us/web/http/cors/index.md:86]: actually `mode` defaults to `"cors"`, so no.

**Pull request merged**